File: python/1043.py
v=input().split(' ')
a=float(v[0])
b=float(v[1])
c=float(v[2])
def check(v1,v2,v3):
    calc=v1+v2
    if calc>v3:
        return True
    else:
        return False
abc=check(a,b,c)
acb=check(a,c,b)
bca=check(b,c,a)
area=((a+b)*c)/2
if abc and acb and bca:
    print('Perimetro =','{:.1f}'.format(a+b+c))
else:
    print('Area =','{:.1f}'.format(area))

File: Java/1043.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.text.DecimalFormat;

public class Main {
    
    static int check(float v1, float v2, float v3) {
        float calc;
        calc = v1 + v2;
        
        if(calc > v3) {
            return 1;
        } else {
            return 0;
        }
    }
    
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        DecimalFormat df = new DecimalFormat("0.0");
        
        float a, b, c, sum, area;
        int abc, acb, bca;
        String[] line = in.readLine().trim().split(" ");
        a = Float.parseFloat(line[0]);
        b = Float.parseFloat(line[1]);
        c = Float.parseFloat(line[2]);
        
        sum = a + b + c;
        area = ((a + b) * c) / 2;
        abc = check(a, b, c);
        acb = check(a, c, b);
        bca = check(b, c, a);
        
        if(abc == 1 && acb == 1 && bca == 1) {
            System.out.println("Perimetro = " + df.format(sum));
        }else {
            System.out.println("Area = " + df.format(area));
        }
    }
}